README.md

```diff
@@ -154,10 +154,13 @@ Define which browsers that ez-build should do its best to target when producing
 
 ```bash
 $ ez-build --target-browsers "last 3 versions"
+$ ez-build --target-browsers "Firefox >=44,Chrome >=48,IE >=9,last 2 version"
```

Comment:
Ok, maybe we should just clarify this with some additional examples at the end of the section then, rather than munging two together like this. I'll take a stab at it.